### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by EPashkin**
cc @GuillaumeGomez @sdroege

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Thanks!

File: src/auto/enums.rs [hunk 1]
```diff
@@ -14,6 +14,7 @@ pub enum ChecksumType {
     Sha256,
     #[cfg(feature = "v2_36")]
     Sha512,
+    Sha384,
```
Comment (sdroege) [inline src/auto/enums.rs]: This is missing a version attribute

Comment (GuillaumeGomez) [reply src/auto/enums.rs]: I didn't notice this one... Thanks for finding it!

Comment (EPashkin) [reply src/auto/enums.rs]: Thanks, @sdroege, I forgot about this

**Pull request merged**